CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Fixed a realm generator issue, when used in concert with MobX. ([#1372](https://github.com/realm/realm-dart/pull/1372))
 * Fix failed assertion for unknown app server errors (Core upgrade, since v12.9.0).
 * Testing the size of a collection of links against zero would sometimes fail (sometimes = "difficult to explain"). (Core upgrade, since v13.15.1)
+* getProgressStream no longer returns a broadcast stream. ([#1375](https://github.com/realm/realm-dart/pull/1375))
```

Comment:
`Sync progress stream returned from Session.getProgressStream() is no longer a broadcast stream`

Perhaps we could be more clear where is that.